packages/cli/src/tools/__tests__/packageManager-test.ts

```diff
@@ -83,9 +100,65 @@ describe('npm', () => {
   });
 });
 
+describe('bun', () => {
```

Comment:
we only test the happy path. what if there's no `bun.lockb` or `getBunVersionIfAvailable` returns false?